sources/packages/web/src/views/student/StudentDashboard.vue

```diff
@@ -6,13 +6,33 @@
     <template #title>
       Welcome {{ user.givenNames }}, let's get started
     </template>
-    <template #content> Please start your Financial Aid application </template>
+    <template #content>
+      Please start your Financial Aid application
+      <br />
+      <br />
+      <Dropdown
```

Comment:
I was expecting that this dropdown would be added to the applications page.
I think that we should point the dashboard to the applications page and add the dropdown there.